lib/components/Keyboard/KeyboardInput/KeyboardAccessoryView.js

```diff
@@ -110,7 +110,15 @@ class KeyboardAccessoryView extends Component {
      *
      * default: false
      */
-    allowHitsOutsideBounds: PropTypes.bool
+    allowHitsOutsideBounds: PropTypes.bool,
+
+    /**
+     * iOS only.
+     * Allow KeyboardAccessoryView to reach to the bottom of the screen (on iPhone x and above)
```

Comment:
1. Doesn't it do the opposite? - Maybe "Set to false in order to allow..."?
2. Is there a new iPhone without SafeArea? (I don't know, maybe SE or something?)

Reply:
`* iOS only.
     * Set false to allow the KeyboardAccessoryView to reach to the bottom of the screen,
     * true to save space to screen's safe area.
     * Supported in all iOS devices that has notch.
     * Other devices (including android) will avoid it and won't have space at the bottom.`